lib/mongo/uri.rb

```diff
@@ -64,6 +65,7 @@ class URI
     # Error details for an invalid scheme.
     #
     # @since 2.1.0
+    # @deprecated
     INVALID_SCHEME = "Invalid scheme. Scheme must be '#{MONGODB_SCHEME}' or '#{MONGODB_SRV_SCHEME}'".freeze
```

Comment:
What is the benefit of keeping this constant in the codebase if it is no longer being used? Documentation purposes?

Reply:
This constant is part of the driver's public API, thus we keep them all around until next major release in case any of the applications make use of them for some reason.